

【日志时间】：2020-06-20 00:08:02,242  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:14:30,319  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:14:52,142  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:15:49,436  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:20:28,741  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:25:17,950  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:33:58,248  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:34:40,915  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:35:18,981  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:41:26,658  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 00:56:19,633  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:01:18,407  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:01:50,734  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:03:17,390  【线程ID】：5    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:04:16,865  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:06:56,629  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:08:21,721  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:10:08,022  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:13:38,026  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:16:21,431  【线程ID】：10    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 01:18:25,170  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 12:06:17,367  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 13:46:48,861  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 13:51:20,443  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 13:54:05,808  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 13:57:07,672  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 14:04:22,484  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:08:37,357  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:10:05,662  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:31:55,514  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:35:01,678  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:39:55,856  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 16:40:06,881  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 17:03:17,611  【线程ID】：63    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.IO.FileNotFoundException: Could not find file 'E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\AllFile\Video\视频1.mp4'.
File name: 'E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\AllFile\Video\视频1.mp4'
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.IO.File.ReadAllBytes(String path)
   at LinXi_CloudStorageApi.Controllers.PictureController.PriviewPic(Int32 UserID, Int32 LxResourceAccountId) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\PictureController.cs:line 72
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 17:22:46,388  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 17:26:46,592  【线程ID】：21    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.IO.FileNotFoundException: Could not find file 'E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\AllFile\Video\视频1.mp4'.
File name: 'E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\AllFile\Video\视频1.mp4'
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.IO.File.ReadAllBytes(String path)
   at LinXi_CloudStorageApi.Controllers.PictureController.PriviewPic(Int32 UserID, Int32 LxResourceAccountId) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\PictureController.cs:line 72
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 17:45:30,936  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 17:46:22,051  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 22:37:04,389  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:23:16,166  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:27:04,663  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:33:41,071  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:33:50,351  【线程ID】：7    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.NullReferenceException: Object reference not set to an instance of an object.
   at LinXi_CloudStorageApi.Controllers.AllFileController.<>c__DisplayClass14_0.<FuzzyQuery>b__1(AllFileDTO a) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 391
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LinXi_CloudStorageApi.Controllers.AllFileController.FuzzyQuery(Int32 UserID, String LxResourceAccountName) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 389
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 23:33:51,500  【线程ID】：7    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.NullReferenceException: Object reference not set to an instance of an object.
   at LinXi_CloudStorageApi.Controllers.AllFileController.<>c__DisplayClass14_0.<FuzzyQuery>b__1(AllFileDTO a) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 391
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LinXi_CloudStorageApi.Controllers.AllFileController.FuzzyQuery(Int32 UserID, String LxResourceAccountName) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 389
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 23:34:29,479  【线程ID】：7    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:36:26,669  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:36:32,928  【线程ID】：12    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.NullReferenceException: Object reference not set to an instance of an object.
   at LinXi_CloudStorageApi.Controllers.AllFileController.<>c__DisplayClass14_0.<FuzzyQuery>b__1(AllFileDTO a) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 397
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LinXi_CloudStorageApi.Controllers.AllFileController.FuzzyQuery(Int32 UserID, String LxResourceAccountName) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 389
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 23:36:45,843  【线程ID】：10    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.NullReferenceException: Object reference not set to an instance of an object.
   at LinXi_CloudStorageApi.Controllers.AllFileController.<>c__DisplayClass14_0.<FuzzyQuery>b__1(AllFileDTO a) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 397
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LinXi_CloudStorageApi.Controllers.AllFileController.FuzzyQuery(Int32 UserID, String LxResourceAccountName) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 389
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)


【日志时间】：2020-06-20 23:37:03,053  【线程ID】：4    
【日志级别】：WARN   
【日志对象】：Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware  
【日志内容】：Failed to determine the https port for redirect. 


【日志时间】：2020-06-20 23:38:55,081  【线程ID】：25    
【日志级别】：ERROR  
【日志对象】：Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware  
【日志内容】：An unhandled exception has occurred while executing the request. 
System.NullReferenceException: Object reference not set to an instance of an object.
   at LinXi_CloudStorageApi.Controllers.AllFileController.<>c__DisplayClass14_0.<FuzzyQuery>b__1(AllFileDTO a) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 397
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at LinXi_CloudStorageApi.Controllers.AllFileController.FuzzyQuery(Int32 UserID, String LxResourceAccountName) in E:\G3项目\LinXi_CloudStorage\LinXi_CloudStorageApi\Controllers\AllFileController.cs:line 389
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
